app/.gitignore

```diff
@@ -1,2 +0,0 @@
-/build
```

Comment:
why are we removing the build directories from each module's gitignore?

Reply:
By executing `./gradlew wrapper --gradle-version=7.4`

Reply:
See https://docs.gradle.org/7.4/release-notes.html

Reply:
They are duplicated, can be handled by the root .gitignore.